editoast/src/views/test_app.rs

```diff
@@ -152,7 +159,7 @@ impl TestAppBuilder {
             infra_caches,
             map_layers: Arc::new(MapLayers::default()),
             speed_limit_tag_ids,
-            disable_authorization: true,
+            disable_authorization: self.disable_authorization,
```

Comment:
This field should have been removed from the `AppState` as it's redundant with the value in `config`. Could we remove it in this PR, it's only used in a few places? 🙏

Reply:
Done.